SampleSwiftMacApp/Cat2CatExampleSwiftMac/Cat2CatExampleSwiftMac/Cat2CatNSImage.swift

```diff
@@ -13,64 +13,74 @@ extension NSImage {
 
     // MARK: - ICONS
     
-    class func ac_AppIcon() -> NSImage? {
-        return NSImage(named:Cat2CatImageNames.AppIcon)
+    class func ac_AppIcon() -> NSImage {
+        return NSImage(c2cName: Cat2CatImageName.AppIcon)
     }
     
-    class func ac_Icon() -> NSImage? {
-        return NSImage(named:Cat2CatImageNames.Icon)
+    class func ac_Icon() -> NSImage {
+        return NSImage(c2cName: Cat2CatImageName.Icon)
     }
     
     // MARK: - Public Domain Icons
     
-    class func ac_No_C() -> NSImage? {
-        return NSImage(named:Cat2CatImageNames.No_C)
+    class func ac_No_C() -> NSImage {
+        return NSImage(c2cName: Cat2CatImageName.No_C)
     }
     
-    class func ac_PD_in_circle() -> NSImage? {
-        return NSImage(named:Cat2CatImageNames.PD_in_circle)
+    class func ac_PD_in_circle() -> NSImage {
+        return NSImage(c2cName: Cat2CatImageName.PD_in_circle)
     }
     
-    class func ac_PDe_Darka_Circle() -> NSImage? {
-        return NSImage(named:Cat2CatImageNames.PDe_Darka_Circle)
+    class func ac_PDe_Darka_Circle() -> NSImage {
+        return NSImage(c2cName: Cat2CatImageName.PDe_Darka_Circle)
     }
     
-    class func ac_SidewaysC() -> NSImage? {
-        return NSImage(named:Cat2CatImageNames.SidewaysC)
+    class func ac_SidewaysC() -> NSImage {
+        return NSImage(c2cName: Cat2CatImageName.SidewaysC)
     }
     
     // MARK: - PHOTOS
     
-    class func ac_Golden_Gate_Bridge() -> NSImage? {
-        return NSImage(named:Cat2CatImageNames.Golden_Gate_Bridge)
+    class func ac_Golden_Gate_Bridge() -> NSImage {
+        return NSImage(c2cName: Cat2CatImageName.Golden_Gate_Bridge)
     }
     
-    class func ac_US_Capitol() -> NSImage? {
-        return NSImage(named:Cat2CatImageNames.US_Capitol)
+    class func ac_US_Capitol() -> NSImage {
+        return NSImage(c2cName: Cat2CatImageName.US_Capitol)
     }
     
-    class func ac_Venice_Beach() -> NSImage? {
-        return NSImage(named:Cat2CatImageNames.Venice_Beach)
+    class func ac_Venice_Beach() -> NSImage {
+        return NSImage(c2cName: Cat2CatImageName.Venice_Beach)
     }
     
-    class func ac_Wrigley_Field() -> NSImage? {
-        return NSImage(named:Cat2CatImageNames.Wrigley_Field)
+    class func ac_Wrigley_Field() -> NSImage {
+        return NSImage(c2cName: Cat2CatImageName.Wrigley_Field)
     }
     
 }
 
-struct Cat2CatImageNames {
+extension NSImage {
+    /**
+    Initializes using the specified Cat2CatImageName.
+    */
+    convenience init(c2cName: Cat2CatImageName) {
+        self.init(named: c2cName.rawValue)!
+    }
+}
```

Comment:
I'm specifically excluding the convenience init for working with IBDesignables since there isn't an equivalent initializer on NSImage.